beater/server.go

```diff
@@ -123,9 +123,14 @@ func createHandler(p processor.Processor, config Config, publish successCallback
 			return
 		}
 
+		err = publish(list)
+		if err != nil {
+			sendError(w, r, 500, fmt.Sprintf(err.Error()), true)
```

Comment:
I suggest to change this to:
```
sendError(w, r, 500, fmt.Sprintf("error publishing events: %s", err.Error()),
```
This will make it easier to find the error in case it is cryptic.